### DISCUSSION TITLE
After Power Outage existing access Key is not able to read/write all folders within a bucket (used for Veeam Backup)

### DISCUSSION THREAD

**Discussion by fsennj** [minio/minio#17313] [Q&A] [answered]
Hello all,

Using Minio in a single host config with 8 disks. all working fine for the last 3 months, yesterday we faced a longer power outage and after this minio did reboot fine but the existing Access Key used for Veeam backup since installation from scratch is not able to read all folders within bucket anymore. 

Minio Version attached. Tried to verify it using an S3 browser so here only 2 folders are shown where as with master User I see 3 folders. How can I repair this please? any help / hint is very much welcome. 

Also did try to set up a new user with access to that bucket after the outage with new access key but that one is also not possible to read/write that 3rd folder.

<img width="324" alt="Bildschirmfoto 2023-05-27 um 20 58 00" src="https://github.com/minio/minio/assets/61477285/1054f13f-470c-4ccc-8056-e0cd71beba16">
<img width="756" alt="Bildschirmfoto 2023-05-27 um 20 59 06" src="https://github.com/minio/minio/assets/61477285/9bf589d0-3cec-4ec3-8793-f0703b3ffde2">
<img width="867" alt="Bildschirmfoto 2023-05-27 um 20 59 55" src="https://github.com/minio/minio/assets/61477285/246e221a-1407-4280-ac93-5d7c8fdd9bc2">

**Comment by klauspost** [comment, marked as answer]
A few checks to go through:

* Check if your file systems are ok.
* Check server log output.
* Check response code of requests. Try performing calls manually, for example with mc.

**Comment by fsennj** [comment]
OK very strange - after several more restarts and upgrading to latest minio binary it started working again.. any clue how to display the status of heal on linux when the original session where the heal was started closed?
